src/main/controllers/IdamSigninController.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+
+export default class IdamSigninController {
+  public get(req: Request, res: Response): void {
+    res.render('idam-signin');
+  }
+
+  public post(req: Request, res: Response): void {
+    const selectChoice = req.body['idam-select'];
+    if (selectChoice == 'crime') {
```

Comment:
please use triple === for comparison as == doesn't compare types hence is considered 'weak' check

Reply:
done